client/vscode/src/chatActivation.ts

```diff
@@ -198,7 +198,7 @@ function generateJS(webView: Webview, extensionUri: Uri): string {
     <script type="text/javascript" src="${entrypoint.toString()}" defer onload="init()"></script>
     <script type="text/javascript">
         const init = () => {
-            amazonQChat.createChat(acquireVsCodeApi());
+            amazonQChat.createChat(acquireVsCodeApi(), {disclaimerAcknowledged: false});
```

Comment:
If I understand this correctly, we now have an expectation that destinations will store this acknowledgement in their local storage. In this case let's be sure to mark the changes in this PR as breaking change and bump versions accordingly and also make sure we talk with toolkit teams. If we were to roll this out now, users would be shown the acknowledge message on each IDE restart

Reply:
True, we need to bump the version.

Reply:
This is a breaking change indeed, as there's a new config field with expectation that destinations will store the acknowledgement state somewhere themselves. The version should be bumped, and I'm in contact with toolkit teams.

Reply:
Not from API, but from behavior point of view. If we push the same major version, clients will get disclaimer popping up again and again until the corresponding event is handled on client side. Which is unacceptable.